project/Dependencies.scala

```diff
@@ -2,8 +2,8 @@ import sbt._
 
 object Dependencies {
   val scalacheckVersion = "1.14.3"
-  val akkaHttpVersion = "10.2.0"
-  val akkaStreamsVersion = "2.6.9"
+  val akkaHttpVersion = "10.1.14"
```

Comment:
Not sure, but isn't this a downgrade of Akka-http?

Reply:
Hi, yes it is. The rest of the stackstate platform was on 10.1.8 or something though. I'm upgrading it to 10.1.14. I was having issues with transitive dependencies etc. that is why I though it best to make it consistent everywhere.